### ISSUE TITLE
"[Proxy] Lost connection to the server" error

### ISSUE THREAD

**Issue by Schiso** [HexagonMC/BungeeCord#140]
### What Hexacord version are you using? Post the output of /bungee
This server is running Proxy version git:BungeeCord-Bootstrap:1.11-SNAPSHOT:40270ea:135 by md_5
Protocol support for 1.7.x by Zartec, ghac and I9hdkill

### Are you using Bungee plugins? If yes, please list all of them + versionnumber
BungeeAuthenticator BPA v5.1 Final3 (latest)
RedisBungee 0.4 (latest) 
Used to ran an 1.7.x / 1.8.x without errors.
### How can we reproduce the error/bug? 
I don't know, it might be something connected to packets of the player

### Is an error message (bungee/spigot/client)? If yes, use pastebin.com and share the link.
Yes, while kicked you get [Proxy] Lost connection to the server.

### In addidtion to that, please explain your issue as detailed as possible.
So basically, sometimes my players get kicked for the error "[Proxy] Lost connection to the server.". They don't get all kicked, time between kicks are 1/20 minutes. (its random)

Thanks!

**Comment by jmcmullen** [comment]
I experienced the same issue and fixed it by using an older release.

**Comment by Schiso** [comment]
Yes, I did the same hahah! Thanks anyway

**Comment by i9hdkill** [comment]
`[Proxy] Lost connection to the server.`
This isn't any useful information. Please check your spigot/forge server log/ client log and bungee log for errors.

**Comment by i9hdkill** [comment]
No info provided.
